SampleShared/Samples/TextPanel/TextPanel.prefab

```diff
@@ -77,7 +77,7 @@ Transform:
   m_GameObject: {fileID: 3954227148511934971}
   m_LocalRotation: {x: 0, y: 0, z: 0, w: 1}
   m_LocalPosition: {x: 0, y: 0, z: 0}
-  m_LocalScale: {x: 1.023, y: 0.174, z: 0.001}
```

Comment:
I'm not sure where this change came from, but it happened from just opening / closing the projects, so I assume this new value is what the scripts on the prefab are defaulting to